volume-providers/vpc/vpcclient/vpcvolume/create_volume.go

```diff
@@ -30,7 +34,10 @@ func (vs *VolumeService) CreateVolume(volumeTemplate *models.Volume) (*models.Vo
 	var volume models.Volume
 	var apiErr models.Error
 
-	_, err := vs.client.NewRequest(operation).JSONBody(volumeTemplate).JSONSuccess(&volume).JSONError(&apiErr).Invoke()
+	request := vs.client.NewRequest(operation)
+	ctxLogger.Info("Equivalent curl command and payload details", zap.Reflect("URL", request.URL()), zap.Reflect("Payload", volumeTemplate))
```

Comment:
Can we add request type as well like POST, GET or QUERY param etc

Reply:
`{"level":"info","ts":"2019-04-18T23:04:05.171+0530","caller":"vpcvolume/create_volume.go:38","msg":"Equivalent curl command and payload details","RequestID":"e521398b-740c-4e1c-bc22-b677079cafc4","URL":"https://us-south.iaas.cloud.ibm.com:443/v1/volumes?version=2019-01-01","Payload":{"name":"simha14","capacity":10,"resource_group":{"id":"efd16c8eac35496e801cf46c19dd03d5"},"tags":["Testing VPC Volume"],"generation":"vpc-classic","profile":{"name":"general-purpose"},"zone":{"name":"us-south-1"}},"Operation":{"Name":"CreateVolume","Method":"POST","PathPattern":"volumes"}}`

Reply:
DONE